src/message.js

```diff
@@ -238,12 +248,18 @@ Message.prototype.encrypt = function(keys, passwords) {
     }
     symEncryptedPacket.packets = this.packets;
 
-    return symEncryptedPacket.encrypt(enums.read(enums.symmetric, symAlgo), sessionKey);
+    return symEncryptedPacket.encrypt(symAlgo, sessionKey);
 
   }).then(() => {
     msg.packets.push(symEncryptedPacket);
     symEncryptedPacket.packets = new packet.List(); // remove packets after encryption
-    return msg;
+    return {
+      message: msg,
+      sessionKey: {
+        data: sessionKey,
+        algorithm: symAlgo
```

Comment:
This is now an enum. I think you should change it back to being a string, personally.